frontend/src/hooks/usePoints.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import {useState} from "react";
+
+export default function usePoints() {
+    const [points, setPoints] = useState(null);
+
+    function getPoints() {
```

Comment:
maybe have a look at react queries they make request handling a bit easier to manage by providing useful helper functions